server/instance.go

```diff
@@ -71,6 +82,39 @@ func (s *Server) registerInstanceRoutes(g *echo.Group) {
 		return nil
 	})
 
+	g.POST("/instance/new-embedded-pg", func(c echo.Context) error {
+		pgUser := "postgres"
+		commonPreffix := "bb-pg"
+		randomSuffix := string(rune(time.Now().Unix()))
+
+		baseDir := fmt.Sprintf("%s/%s_%s", common.GetPostgresSocketDir(), commonPreffix, randomSuffix)
+		resourceDir := path.Join(baseDir, "resources")
+		dataDir := path.Join(baseDir, "pgdata")
+
+		pgInstance, err := postgres.Install(resourceDir, dataDir, pgUser)
```

Comment:
nit: this will unzip the PostgreSQL binary plus initialize the data directory. We may only want the latter one but use the existing PostgreSQL binary.

Reply:
Updated.